app/views/events/_table_badges.html.erb

```diff
@@ -1,30 +1,253 @@
+    <!-- Modal -->
+    <div id="table_<%= table.id %>_core" class="modal fade" role="dialog">
+      <div class="modal-dialog">
+        <!-- Modal content-->
+        <div class="modal-content">
+          <div class="modal-header">
+            <button type="button" class="close" data-dismiss="modal">&times;</button>
+            <h4 class="modal-title">Core Campaign</h4>
+          </div>
+          <div class="modal-body">
+            <p>This table is being offered as a Core Campaign game. You must have a Core Campaign character in order to
+              play, unless you would like play a Core pregen.</p>
+          </div>
+          <div class="modal-footer">
+            <button type="button" class="btn btn-default" data-dismiss="modal">Close</button>
+          </div>
+        </div>
+      </div>
+    </div>
 <% end %>
 <% if table.non_pfs? %>
-    <span class="badge badge-non-pfs">No Society Credit</span>
+    <button type="button" class="btn btn-round-xs btn-non-pfs btn-xs" data-toggle="modal" data-target="#table_<%= table.id %>_nonpfs">No
+      Society Credit
+    </button>
+    <!-- Modal -->
+    <div id="table_<%= table.id %>_nonpfs" class="modal fade" role="dialog">
+      <div class="modal-dialog">
+        <!-- Modal content-->
+        <div class="modal-content">
+          <div class="modal-header">
+            <button type="button" class="close" data-dismiss="modal">&times;</button>
+            <h4 class="modal-title">No Society Credit</h4>
+          </div>
+          <div class="modal-body">
+            <p>This scenario is not being run for organized play credit. You will not be receiving a chronicle for
+              playing. It may be pregen only, with Society characters, or otherwise. See "More Info!" for more
+              details.</p>
+          </div>
+          <div class="modal-footer">
+            <button type="button" class="btn btn-default" data-dismiss="modal">Close</button>
+          </div>
+        </div>
+      </div>
+    </div>
 <% end %>
 <% if table.raffle? %>
-    <span class="badge badge-raffle">Raffle</span>
+    <button type="button" class="btn btn-round-xs btn-raffle btn-xs" data-toggle="modal" data-target="#table_<%= table.id %>_raffle">Raffle</button>
+    <!-- Modal -->
+    <div id="table_<%= table.id %>_raffle" class="modal fade" role="dialog">
+      <div class="modal-dialog">
+        <!-- Modal content-->
+        <div class="modal-content">
+          <div class="modal-header">
+            <button type="button" class="close" data-dismiss="modal">&times;</button>
+            <h4 class="modal-title">Raffle</h4>
+          </div>
+          <div class="modal-body">
+            <p>This table will be offered as a raffle during the event. You will need purchase raffle tickets, which
+              will be drawn prior to the session. See "More Info!" for mode details on when the raffle will happen, and
+              for the raffle ticket price.</p>
+          </div>
```

Comment:
Is this still valid as a real option? Does it need this text or will this eventually be removed?